src/components/changePassword/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+
+import { isEmpty } from 'lodash';
```

Comment:
The lodash size is about 72 kb. Is it necessary to use this library in your code? 

Maybe `Object.keys(errors).length > 0`?

Reply:
!

Reply:
In the feature, we will change webpack config for resolving this problem.
https://github.com/GoogleChromeLabs/webpack-libs-optimizations#lodash